cfme/automate/explorer/method.py

```diff
@@ -228,6 +229,37 @@ def read(self):
         return self.all_vars
 
 
+class EntryPoint(Text, ClickableMixin):
+    def fill(self, value):
+        """Used to select the automate method from automate domain"""
+        if value:
+            self.click()
+            self.parent_view.embedded_view.wait_displayed('15s')
+            self.parent_view.embedded_view.tree.click_path(*value)
+            return True
+        return None
+
+
+class EmbeddedView(View):
```

Comment:
It looks to me like this should potentially use the wt.pf `Modal` view, which provides several properties and methods for interacting with the modal.

Please evaluate using it, keeping in mind that if `ROOT` locator doesn't exactly match, it can easily be changed in this local subclass.